authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

```diff
@@ -54,7 +54,11 @@ The credential you provide for mint mode in Google Cloud Platform (GCP) must hav
 * `iam.serviceAccounts.create`
 * `iam.serviceAccounts.delete`
 * `iam.serviceAccounts.get`
+* `iam.roles.create`
 * `iam.roles.get`
+* `iam.roles.list`
+* `iam.roles.undelete`
+* `iam.roles.update`
```

Comment:
**Mint mode**

These lines are the set from [here](https://issues.redhat.com/browse/OSDOCS-4157?focusedId=23744933&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23744933). Are all the other ones superfluous?

Reply:
These are in addition to the previously existing permissions.